### PR TITLE
Adds in a bunch of new random maintenance rooms

### PR DESCRIPTION
**Pull request by matskuman5**
This is made possible by #2122 , thanks to Cruix. Many ideas from Dagdammit.

Adds in 6 new random maintenance rooms in 2 different places. These rooms contain some loot that is a bit more powerful than stuff you'd normally find in maintenance. To balance that out you don't know what you'll get.

You can take a look at the maps if you really want to spoil yourself (or if you're balance police)

More suggestions are always welcome

:cl:
rscadd: Adds in a bunch of new randomly selected rooms that replace some old, unused rooms in maintenance. Credit to Cruix for the framework and to Dagdammit for most of the ideas.
/:cl:

### CODE REVIEW

**Comment by Zostroll** [comment]
Could we get some pictures?

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 1]
```diff
@@ -99,6 +99,17 @@
 	for(var/i in 1 to 3)
 		new /obj/item/weapon/reagent_containers/blood/random(src)
 
+/obj/structure/closet/crate/freezer/xeno
+	name = "freezer"
+	desc = "It smells... funky."
+
+/obj/structure/closet/crate/freezer/New()
```
Comment (Cruix) [inline code/game/objects/structures/crates_lockers/crates.dm]: freezer/xeno/New(), or the chef will have a weird day

File: code/modules/paperwork/paper.dm [hunk 2]
```diff
@@ -392,6 +392,14 @@
 	name = "paper- Smelting Operations Closed"
 	info = "<B>**NOTICE**</B><BR><BR>Smelting operations moved on-station.<BR><BR>Take your unrefined ore to the Redeption Machine in the Delivery Office to redeem points.<BR><BR>--SS13 Command"
 
+/obj/item/weapon/paper/druglab
+	name = "Security Notice"
+	info = "<small> On September 20th, 2551 </small> <br> <br> Janitor: Don't touch this until the detective's given you the all clear. We don't want to mess the evidence up now that we've catched the most notorious drug baron this side of Tau Ceti."
```
Comment (Cruix) [inline code/modules/paperwork/paper.dm]: caught

**Comment by matskuman5** [comment]
Right, fixed what you pointed out

**Comment by Super3222** [comment]
The code looks good.

**Pull request merged**